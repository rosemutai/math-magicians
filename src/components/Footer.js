const Footer = () => (
  <div className="footer w-full absolute py-8 text-center bottom-0 bg-gray-100 font-smibold">
    <p className="">Math Magicians &copy;2023</p>
  </div>
);

export default Footer;
