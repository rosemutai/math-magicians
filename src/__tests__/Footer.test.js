import { render } from '@testing-library/react';
import Footer from '../components/Footer';

describe('Footer', () => {
  it('renders Footer component correctly', () => {
    const { container } = render(
      <Footer />,
    );
    expect(container).toMatchSnapshot();
  });
});
