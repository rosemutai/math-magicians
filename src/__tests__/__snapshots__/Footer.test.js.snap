// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer component correctly 1`] = `
<div>
  <div
    class="footer w-full absolute py-8 text-center bottom-0 bg-gray-100 font-smibold"
  >
    <p
      class=""
    >
      Math Magicians Â©2023
    </p>
  </div>
</div>
`;
